tidb/src/main/java/io/tidb/bigdata/tidb/allocator/RowIDAllocator.java

```diff
@@ -133,24 +133,27 @@ private void set(@Nonnull List<BytePairWrapper> pairs, @Nonnull TiTimestamp time
     if (!iterator.hasNext()) {
       return;
     }
-    TwoPhaseCommitter twoPhaseCommitter = new TwoPhaseCommitter(session, timestamp.getVersion());
-    BytePairWrapper primaryPair = pairs.get(0);
-    twoPhaseCommitter.prewritePrimaryKey(
-        ConcreteBackOffer.newCustomBackOff(2000), primaryPair.getKey(), primaryPair.getValue());
-
-    if (!iterator.hasNext()) {
-      twoPhaseCommitter.prewriteSecondaryKeys(primaryPair.getKey(), iterator, 20000);
-    }
+    TwoPhaseCommitter twoPhaseCommitter =
+        new TwoPhaseCommitter(session, timestamp.getVersion(), 1000L);
+    try {
+      BytePairWrapper primaryPair = pairs.get(0);
+      twoPhaseCommitter.prewritePrimaryKey(
+          ConcreteBackOffer.newCustomBackOff(1000), primaryPair.getKey(), primaryPair.getValue());
 
-    twoPhaseCommitter.commitPrimaryKey(
-        ConcreteBackOffer.newCustomBackOff(5000),
-        primaryPair.getKey(),
-        session.getTimestamp().getVersion());
+      if (!iterator.hasNext()) {
+        twoPhaseCommitter.prewriteSecondaryKeys(primaryPair.getKey(), iterator, 1000);
+      }
 
-    try {
-      twoPhaseCommitter.close();
-    } catch (Throwable ignored) {
-      // ignore
+      twoPhaseCommitter.commitPrimaryKey(
+          ConcreteBackOffer.newCustomBackOff(1000),
+          primaryPair.getKey(),
+          session.getTimestamp().getVersion());
+    } finally {
+      try {
+        twoPhaseCommitter.close();
+      } catch (Throwable ignored) {
+        // ignore
```

Comment:
better to print log

Reply:
It is copied from tispark, but it would be butter to print error log.